### PR TITLE
[digitalstrom] code refactoring

### PR DESCRIPTION
**Pull request by lsiepel**
- Remove apache.commons
 - Minor checkstyle fixes

Untested, only changes from code perspective.

### CODE REVIEW

File: bundles/org.openhab.binding.digitalstrom/src/main/java/org/openhab/binding/digitalstrom/internal/lib/structure/devices/deviceparameters/impl/DSID.java [hunk 1]
```diff
@@ -35,7 +35,7 @@ public DSID(String dsid) {
         if (trimmedDsid.length() == 24) {
             this.dsid = trimmedDsid;
         } else if (trimmedDsid.length() == 8) {
-            this.dsid = this.PRE + trimmedDsid;
+            this.dsid = DSID.PRE + trimmedDsid;
```
Comment (wborn) [inline bundles/org.openhab.binding.digitalstrom/src/main/java/org/openhab/binding/digitalstrom/internal/lib/structure/devices/deviceparameters/impl/DSID.java:38]:
```suggestion
            this.dsid = PRE + trimmedDsid;
```

**Review by wborn — CHANGES_REQUESTED** [review]
Thanks for the cleanup! I think there is a small improvement:

File: bundles/org.openhab.binding.digitalstrom/src/main/java/org/openhab/binding/digitalstrom/internal/lib/structure/devices/deviceparameters/impl/DSID.java (hunk 1 above)
Comment (lsiepel) [reply bundles/org.openhab.binding.digitalstrom/src/main/java/org/openhab/binding/digitalstrom/internal/lib/structure/devices/deviceparameters/impl/DSID.java]: Fixed

**Review by wborn — APPROVED** [review]
👍

**Pull request merged**